app/src/main/java/ui/settings/about/PersonAdapter.kt

```diff
@@ -109,80 +75,137 @@ class PersonAdapter internal constructor() : BaseRecyclerViewAdapter(), KoinComp
         return HeaderHolder(textView)
     }
 
-    override fun getItemViewType(position: Int): Int = if (items[position] is Person) PERSON else -1
+    override fun getItemViewType(position: Int): Int =
+        if (items[position] is Person) VIEW_TYPE_PERSON else -1
 
     override fun getItemCount(): Int = items.size
 
     /**
      * Header in the list
      */
     internal inner class HeaderHolder(private val view: TextView) :
-            BaseRecyclerViewAdapter.BaseHolder(view) {
+        BaseRecyclerViewAdapter.BaseHolder(view) {
 
         override fun bind(position: Int) {
-            view.setText(items[position] as Int)
+            val title = items[position] as? Int ?: return
```

Comment:
You may consider this a safer binding, but having the item not be an int is unexpected behaviour

Reply:
I'm not sure what you're getting at here. Would you rather error out if this is the case, i.e. 
```suggestion
            val title = items[position] as? Int ?: error("Not an Int")
```

Reply:
Given previous checks, you should never have a viewholder with an incorrectly associated data type. This way of handling prevents a crash, but doesn't actually give a desired solution when the cast fails, as it'll mess up the UI.

You can leave this for now, but in the future we can look into creating a different type of adapter where the layoutRes, viewholder, and data are all held in a single model. That way you can deal with whatever data you wish without needing to cast. An example of such an adapter is [FastAdapter](https://github.com/mikepenz/FastAdapter), but you can write your own if you don't want to import another library.

Reply:
Fair enough. I'll rework the adapters I have in Suitcase hold those three pieces of data together.